scripts/variantstore/wdl/GvsQuickstartHailIntegration.wdl

```diff
@@ -124,17 +111,125 @@ task CreateAndTieOutVds {
 
         python3 ./hail_gvs_import.py \
             --avro-path ${AVRO_PATH} \
-            --vds-path ${VDS_PATH} \
+            --vds-path ~{vds_destination_path} \
             --temp-path ${TEMP_PATH} \
             --references-path ${REFERENCES_PATH} \
             ~{true='' false='--use-vqsr-lite' use_classic_VQSR}
 
-        export JOINED_MATRIX_TABLE_PATH=${WORK}/joined.mt
+    >>>
+    runtime {
+        # `slim` here to be able to use Java
+        docker: "gcr.io/google.com/cloudsdktool/cloud-sdk:426.0.0-slim" ## should this be our regular docker now?
+        disks: "local-disk 2000 HDD"
+        memory: "30 GiB"
+    }
+    output {
+        Boolean done = true
+        ## TODO should this optionally pass the VDS or just put it in gcp?
+        ## maybe change this to:         String vds_path = vds_destination_path
+    }
+}
+
+task ValidateVds {
+    input {
+        String vds_path
+    }
+
+    command <<<
+        # Prepend date, time and pwd to xtrace log entries.
+        PS4='\D{+%F %T} \w $ '
+        set -o errexit -o nounset -o pipefail -o xtrace
+
+        # Copy the validation script from GitHub-- TODO where should this really live?
+        curl --silent --location --remote-name "https://raw.githubusercontent.com/broadinstitute/gatk/ah_var_store/scripts/variantstore/docs/aou/vds/vds_validation.py"
```

Comment:
again making me wonder if we want a 'slim' + Java + Hail + our scripts image for this Hail stuff...